samples/listctrl/listtest.cpp

```diff
@@ -1229,13 +1229,6 @@ void MyListCtrl::OnListKeyDown(wxListEvent& event)
 {
     long item;
 
-    if ( !wxGetKeyState(WXK_SHIFT) )
```

Comment:
@vadz,
Because with those lines you can't check the size properly - pressing 1, 2, 3 or 4 will not produce the result of sub item rectangle, which is retrieved later on. And of course you can't test the Ctrl+1{,2,3,4}, which the sample tests later on.
Did you try the code in the here with and without the patch?

I am not sure why this code is here in the first place - but I can place the comment there why it has to be removed.

Let me know.

Reply:
This code is useful for testing key events in the control. You could just remove the `return` or use `Shift+N` for testing the other stuff instead.

Reply:
@vadz,
Please look at the code - all 3 combinations of "2", "Ctrl+2" and "Shift+2" are used. Its to check item, icon and label bounding box.
I am happy to just remove the return, 

Please advise.